geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-parquet/src/main/scala/org/locationtech/geomesa/fs/storage/parquet/io/SimpleFeatureReadSupport.scala

```diff
@@ -179,6 +190,17 @@ object SimpleFeatureReadSupport {
     }
   }
 
+  private def geometryV2(binding: ObjectType, i: Int, callback: Settable): Converter = {
+    val geomTypes = Seq(ObjectType.POINT, ObjectType.LINESTRING, ObjectType.POLYGON, ObjectType.MULTIPOINT,
+      ObjectType.MULTILINESTRING, ObjectType.MULTIPOLYGON, ObjectType.GEOMETRY)
+
+    if (geomTypes.contains(binding)) {
```

Comment:
don't think we need this check here, this method is only called right after checking the object type.